ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetProtocolSpecFactory.java

```diff
@@ -207,8 +207,20 @@ public ProtocolSpecBuilder cancunDefinition(final GenesisConfigOptions genesisCo
         evmConfiguration);
   }
 
-  public ProtocolSpecBuilder shandongDefinition(final GenesisConfigOptions genesisConfigOptions) {
-    return MainnetProtocolSpecs.shandongDefinition(
+  public ProtocolSpecBuilder futureDefinition(final GenesisConfigOptions genesisConfigOptions) {
```

Comment:
Might be good to javadoc how to decide whether features should go under future or experimental?